toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -802,7 +813,12 @@ func (b *LiveOSIsoBuilder) prepareLiveOSDir(inputSavedConfigsFilePath string, wr
 		return err
 	}
 
-	b.artifacts.dracutPackageInfo, err = getDracutVersion(writeableRootfsDir)
+	b.artifacts.dracutPackageInfo, err = getPackageInformationFromRootfsDir(writeableRootfsDir, "dracut")
+	if err != nil {
+		return err
+	}
+
+	b.artifacts.selinuxPolicyPackageInfo, err = getPackageInformationFromRootfsDir(writeableRootfsDir, "selinux-policy")
```

Comment:
MIC supports the user using their own custom SELinux policies. (See, UpdateSELinuxModeInConfigFile.) If the user is using the `selinux-policy` package, then we should do the version check. Otherwise, assume the user is using a compatible policy.

Reply:
pending

Reply:
fixed.